internal/pkg/template/templates/workloads/services/backend/manifest.yml

```diff
@@ -34,6 +34,23 @@ image:
     start_period: {{.ImageConfig.HealthCheck.StartPeriod}}
 {{- end}}
 
+{{- if gt (len .HTTP.AdditionalRoutingRules) 0}}
```

Comment:
Do we need this? I thought we won't prompt users to add ports for backend service.

Reply:
we do not prompt users to add ports for backend service but if customers do `copilot svc init --port 3000,3001` and then choose backend service then we are sure that customer knows about multiple ports concept and they are doing conscious effort to add  internal LB that too with multiple ports. At least that made sense to me unless you have another opinion.

Reply:
I like this, I just want to make sure that we have a description telling them to check their manifest and customize the paths for each port for backend services. I'll add that in my next commit if something like it doesn't exist.